bmemcached/client/mixin.py

```diff
@@ -30,14 +30,16 @@ def __init__(self, servers=('127.0.0.1:11211',), username=None,
                  socket_timeout=SOCKET_TIMEOUT,
                  pickle_protocol=0,
                  pickler=pickle.Pickler,
-                 unpickler=pickle.Unpickler):
+                 unpickler=pickle.Unpickler,
+                 tls_context=None):
```

Comment:
Hey there, could you also add some info about in on the __doc__?

Reply:
Sure, I'm on vacation now and will be back on this task in the first week of January.